packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -54,7 +56,8 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
       async (t, c) => {
         const {prepareProject} = await import('./project-setup/index');
         return prepareProject(options)(t, c);
-      }
+      },
+      displayModuleList(OTTER_MODULE_KEYWORD, OTTER_MODULE_SUPPORTED_SCOPES)
```

Comment:
improvement suggestion:
Could we add a new entry in the generators of core package to list the available otter modules, without running the `ng add @o3r/core` and see them at the end?
something like `npx ng g @o3r/core:show-modules` ?

Reply:
Added